### DISCUSSION TITLE
DataGrid is not available in UNO platform project.

### DISCUSSION THREAD

**Discussion by ApoorvaPetkar** [unoplatform/uno#9384] [Q&A] [answered]
Hi 

I'm not able to find DataGrid component in the UNO project created which is using Microsoft.UI.Xaml.Controls for controls.
I see it's available in the Windows community toolkit, but I'm not sure if it works for cross-platform as I'm trying to build cross-platform app using UNO( https://platform.uno/docs/articles/get-started-vs-2022.html?tabs=windows)

Please help me clarify with your inputs.

Thanks in advance.

**Comment by francoistanguay** [comment, marked as answer]
The WCT one is the datagrid that Uno supports indeed.

**Reply by jeromelaban** [reply]
You can find how to use the Windows Community Toolkit [here](https://platform.uno/docs/articles/uno-community-toolkit.html?tabs=tabid-winui), as well as a DataGrid sample [here](https://github.com/unoplatform/Uno.Samples/tree/master/UI/UnoWCTDataGridSample).

**Comment by ApoorvaPetkar** [comment]
Thank you it worked
